start.sh

```diff
@@ -31,5 +31,7 @@ echo 'setting environment variables ...'
 eval "$(./boot2docker.exe shellinit 2>/dev/null | sed  's,\\,\\\\,g')"
 echo
 
+echo 'you can now use docker commands directly, or `boot2docker ssh` to log into the Boot2Docker VM'
```

Comment:
+1; the verbiage sounds fine to me, too

I'd say we should probably also put `docker` in backticks for consistency, and maybe start with an uppercase letter and end with a period. :+1: